packages/twenty-front/src/modules/settings/data-model/objects/SettingsObjectCoverImage.tsx

```diff
@@ -8,11 +8,15 @@ import { Card } from '@/ui/layout/card/components/Card';
 import { AnimatedFadeOut } from '@/ui/utilities/animation/components/AnimatedFadeOut';
 import { cookieStorage } from '~/utils/cookie-storage';
 
-import CoverImage from '../assets/cover.png';
+import DarkCoverImage from '../assets/cover-dark.png';
+import LightCoverImage from '../assets/cover-light.png';
 
 const StyledCoverImageContainer = styled(Card)`
   align-items: center;
-  background-image: url(${CoverImage.toString()});
+  background-image: ${({ theme }) =>
+    theme.name === 'light'
```

Comment:
Just wondering if we should change theme.ts and align with the backend to avoid potential bugs, it seems we store ColorScheme.Light with 'Light' in DB and not 'light'.

Reply:
cc @charlesBochet